client/src/components/App.js

```diff
@@ -24,10 +27,12 @@ const App = (props) => {
       <TopBar user={currentUser} />
       <Switch>
         <Route exact path="/">
-          <h2>Hello from react</h2>
+          <YelpRequestForm />
         </Route>
```

Comment:
For consistency this can be refactored to follow the same Route pattern you have below